addons/skin.estouchy/xml/DialogPVRInfo.xml

```diff
@@ -198,6 +198,13 @@
 				<label>19165</label>
 				<visible>Window.IsActive(PVRGuideInfo)</visible>
 			</control>
+			<control type="button" id="9">
+				<description>RePlay Program</description>
```

Comment:
Again, no captial P in Replay. 
Additionally, not "Program", but "program" (Kodi UI design guideline).
I'm not sure whether 'program' is the right word here. We use "show" elsewhere.